cmd/gaia/testnets/README.md

```diff
@@ -110,7 +81,7 @@ Your node needs to know how to find peers. You'll need to add healthy seed nodes
 
 ```toml
 # Comma separated list of seed nodes to connect to
-seeds = "38aa9bec3998f12ae9088b21a2d910d19d565c27@gaia-6002.coinculture.net:46656,1e124dd15bd9955a7ea844ab003b1b47f0998b70@seed.cosmos.cryptium.ch:46656"
+seeds = "718145d422a823fd2a4e1e36e91b92bb0c4ddf8e@gaia-7000.coinculture.net:26656,5922bf29b48a18c2300b85cc53f424fce23927ab@67.207.73.206:26656,7c8b8fd03577cd4817f5be1f03d506f879df98d8@gaia-7000-seed1.interblock.io,a28737ff02391a6e00a1d3b79befd57e68e8264c@gaia-7000-seed2.interblock.io,987ffd26640cd03d08ed7e53b24dfaa7956e612d@gaia-7000-seed3.interblock.io"
```

Comment:
gaiad complains that the `*interblock.io` seeds do not have port numbers:
ERROR: Error{Invalid address (gaia-7000-seed1.interblock.io): address gaia-7000-seed1.interblock.io: missing port in address}